bin/pr_reviewers_ping.dart

```diff
@@ -1,6 +1,19 @@
+import 'package:args/args.dart';
 import 'package:pr_reviewers_ping/pinger.dart' as pr_reviewers_ping;
 
+const _tokenArgument = 'token';
+const _skipNoPrsMessageArgument = 'skip_no_prs_message';
+
 void main(List<String> arguments) {
-  final skipNoPrsMessage = arguments.contains('--skip_no_prs_message');
-  pr_reviewers_ping.Pinger().pingPullRequestReviewers(skipNoPrsMessage: skipNoPrsMessage);
+  final parser = ArgParser();
+
+  parser.addOption(_tokenArgument, mandatory: true);
```

Comment:
This way token from Github actions can be used, so no need to use personal access token in config anymore